docs/UsingConfigManager/UsingConfigManager.xml

```diff
@@ -706,6 +706,67 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
       environment XML file.</para>
 
       <sect2 role="brk">
+        <title>Backup Node</title>
+
+        <para>Backup Node allows you to back up Thor clusters at regular
+        intervals. The Backup Node component is a way to allow administrators
+        to manage the backup node process without using a cron job.</para>
+
+        <para>To enable back up node, you add the backupnode component, choose
+        the hardware instance to run it on and then add Thor groups to it.
+        <graphic fileref="images/CM-img21.jpg" vendor="configmgrSS" /></para>
+
+        <para><orderedlist>
+            <listitem>
+              <para>Right click on the <emphasis
+              role="bold">Software</emphasis> component in the <emphasis
+              role="bold">Navigator</emphasis> panel (on the right side),
+              choose <emphasis role="bold">New Components</emphasis> then
+              <emphasis role="bold">backupnode</emphasis> from the drop
+              list.</para>
+            </listitem>
+
+            <listitem>
+              <para>From the tabs on the right side, select the <emphasis
+              role="bold">Instances</emphasis> tab.</para>
+            </listitem>
+
+            <listitem>
+              <para>Right click on the <emphasis
+              role="bold">computer</emphasis> column and choose <emphasis
+              role="bold">Add Instances</emphasis>...<graphic
+              fileref="images/CM-img22.jpg" vendor="configmgrSS" /></para>
+            </listitem>
+
+            <listitem>
+              <para>Select the computer for the backupnode component, or press
```

Comment:
Which server would I choose in a large system? ThorMaster? One of the support nodes?